sys-apps/irqbalance/files/irqbalance.service

```diff
@@ -2,10 +2,8 @@
 Description=CPU Interrupt Balancer
 
 [Service]
-ExecStart=/usr/sbin/irqbalance --debug
-# --debug implies a lot of cruft
-# but it's the only way to get foregrounding
-StandardOutput=null
+EnvironmentFile=-/etc/irqbalance.env
```

Comment:
It's not really 'upstream standard location' but a hack made when I pointed out they use Fedora-specific location. And yes, in Gentoo we don't approve such locations but use standard systemd `service.d` files.